www-servers/automx2/automx2-2019.0.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="Email client autoconfiguration service"
+HOMEPAGE="https://automx.org/"
+SRC_URI="https://gitlab.com/automx/automx2/-/archive/${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+
+BDEPEND="acct-user/automx2
+	$(python_gen_cond_dep \
+		'>=dev-python/flask-migrate-2.5.2[${PYTHON_MULTI_USEDEP}]' python3_{7}
+	)"
+RDEPEND="${BDEPEND}"
+
+python_prepare_all() {
+	sed -i -e "/('scripts'/d" setup.py || die
+	distutils-r1_python_prepare_all
+}
+
+python_test() {
+	export AUTOMX2_CONF="tests/unittest.conf"
+	${EPYTHON} -m unittest discover tests/ || die
+}
+
+python_install_all() {
+	sed -e "s/@EPYTHON@/${EPYTHON}/" "${FILESDIR}/init" | newinitd - "${PN}"
```

Comment:
You could call the sedding in `src_prepare` where it belongs. And add corresponding `|| die` that our QA team loves so much (if I remember correctly, there's a policy that all external commands should die if they fail to execute, but hey you're much more familiar with searching these all-around scattered policies currently)

Reply:
This sed-pipe-newinitd variant relies on not requiring a temporary file, so I can only invoke it at the time newinitd is executed. The `die` however is missing, thanks for spotting that.